src/components/about-token.js

```diff
@@ -0,0 +1,135 @@
+import React from 'react'
+import { FormattedMessage } from 'react-intl'
+
+const AboutToken = () => {
+  return (
+    <div className="about-token-wrapper">
+      <div className="container">
+        <div className="row">
+          <div className="col-md-6">
+            <h1>
+              <FormattedMessage
+                id={'about-token.about'}
+                defaultMessage={'About Origin Tokens'}
+              />
+            </h1>
+            <h3 className="lead">
+              <FormattedMessage
+                id={'about-token.use'}
+                defaultMessage={'What are Origin Tokens used for?'}
+              />
+            </h3>
+            <h6><strong>
```

Comment:
@mek32390 these opening and closing `strong` tags should be nested equally for readability.